src/framework/CMakeLists.txt

```diff
@@ -223,19 +223,25 @@ message(STATUS "LuaJIT: " ${LUAJIT})
 
 find_package(PhysFS REQUIRED)
 find_package(ZLIB REQUIRED)
+find_package(Protobuf REQUIRED)
+find_package(LibLZMA REQUIRED)
 
 set(framework_LIBRARIES ${framework_LIBRARIES}
     ${Boost_LIBRARIES}
     ${LUA_LIBRARY}
     ${PHYSFS_LIBRARY}
     ${ZLIB_LIBRARY}
+    ${PROTOBUF_LIBRARY}
+    ${LIBLZMA_LIBRARY}
```

Comment:
I believe this should be LIBLZMA_LIBRARIES, at least on my system LIBLZMA_LIBRARY is empty.

Reply:
It compiled in my ubuntu like this. I'll try what you say, and if it compiles aswell I'll change it. Looks like you have more knowledge than I do with those CMakeLists files 😂